releases/20.0.0.9/full/Dockerfile.ubi.ibmjava8

```diff
@@ -1,4 +1,5 @@
 FROM openliberty/open-liberty:20.0.0.9-kernel-java8-ibmjava-ubi
+ARG VERBOSE=false
 
 RUN cp /opt/ol/wlp/templates/servers/javaee8/server.xml /config/server.xml
 
```

Comment:
@ymanton just to confirm, IBM java doesn't need updates to ENV IBM_JAVA_OPTIONS?

Reply:
Currently no, it doesn't. The IBM SDK image doesn't ship with an SCC so we should continue to build a Liberty SCC in that case.